README.md

```diff
@@ -52,10 +72,30 @@ pip install -e .  --no-dependencies
 
 See `pyproject.toml` for a list of dependencies tied to each subpackage.
 
+### Model checkpoints
+
+See table below for links to model checkpoints trained on USPTO-50K alongside with information on how these checkpoints were obtained.
+Note that all checkpoints were produced in a way that involved external model repositories, hence may be affected by the exact license each model was released with.
+For more details about a particular model see the top of the corresponding model wrapper file in `reaction_prediction/inference/`.
+
+
+| Model checkpoint link                                          | Source |
```

Comment:
I just moved it here from the other file 😅

I'm also considering to later on move these links to e.g. a `*.yml` file so that they are available programmatically; then we can have the model wrappers automatically use the USPTO-50K checkpoint if no path is provided. That would make setup even quicker as one wouldn't have to manually download the checkpoint.